<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QPushButton" name="sendBtn">
      <property name="text">
       <string>Отправить</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Отправляемая команда:</string>
      </property>
      <property name="buddy">
       <cstring>inputCmd</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="inputCmd"/>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="settingsBtn">
      <property name="text">
       <string>Настройки сервера</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Полученная команда:</string>
      </property>
      <property name="buddy">
       <cstring>displayCmd</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTextBrowser" name="displayCmd"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>displayCmd</tabstop>
  <tabstop>inputCmd</tabstop>
  <tabstop>sendBtn</tabstop>
  <tabstop>settingsBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
